src/test/java/com/marklogic/migration/rdbms/SingleQueryTest.java

```diff
@@ -6,7 +6,7 @@
 
 	@Test
 	public void test() {
-		String migrationJson = "{\"query\":\"select * from Customer where customer_id=1\"," +
+		String migrationJson = "{\"query\":\"select * from customer where customer_id=1\"," +
```

Comment:
I had to use lower case table names to get the SQL queries to work with the sakila dataset I downloaded today